cmd/query/app/querysvc/query_service_v2.go

```diff
@@ -0,0 +1,141 @@
+// // Copyright (c) 2019 The Jaeger Authors.
+// // SPDX-License-Identifier: Apache-2.0
+
+package querysvc
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+	"go.uber.org/zap"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/model/adjuster"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+	"github.com/jaegertracing/jaeger/storage_v2/depstore"
+	"github.com/jaegertracing/jaeger/storage_v2/tracestore"
+)
+
+// TODO: Remove query_service.go and rename query_service_v2.go
+// to query_service.go once all components have been migrated to
+// operate on the OTEL data model.
+var errNoArchiveSpanStorageV2 = errors.New("archive span storage was not configured")
+
+const (
+	defaultMaxClockSkewAdjustV2 = time.Second
+)
```

Comment:
rather than redefining I would move them here from query_service.go and reuse the names. We don't have to duplicate what we don't need to.